openforcefield/typing/engines/smirnoff/forcefield.py

```diff
@@ -1326,8 +1326,18 @@ def create_openmm_system(self, topology, **kwargs):
 
         # Let force Handlers do postprocessing
         for parameter_handler in parameter_handlers:
+            if parameter_handler._TAGNAME == "vdW":
+                continue
             parameter_handler.postprocess_system(system, topology, **kwargs)
 
+        electrostatics_14 = self.get_parameter_handler(tagname="Electrostatics").scale14
+        # Absolute hack to get around the fact that the vdW handler needs to know the
+        # Electrostatics's 1-4 scaling factor; maybe this code should be copied directly
+        # here, instead of being in a postprocess_system?
+        self._parameter_handlers["vdW"].postprocess_system(
+            system, topology, electrostatics_14
+        )
```

Comment:
Oof. This is a tough call, but yeah, let's move all the `CreateExceptionsFromBonds` stuff up to `ForceField.create_openmm_system`

This will add complexity, but we can't sidestep the underlying incongruence between us having separate ParameterHandlers that map to a single OpenMM `NonbondedForce`. So the complexity has to live somewhere, and at least we can put it all in one place here. 

This got me thinking -- If the FF is missing an ElectrostaticsHandler or vdWHandler, what should we do? Ideas are either
* add an informative error apologizing and saying that we can't do it
* Assume that 1-4 scaling factors from absent handlers are 0

I'm slightly in favor of the "informative error" but could easily be convinced otherwise.